modules/publisher/views.py

```diff
@@ -254,7 +254,7 @@ def get_market_snap(snap_name):
     context = {
         "snap_id": snap_details['snap_id'],
         "snap_name": snap_details['snap_name'],
-        "title": snap_details['title'],
+        "snap_title": snap_details['title'],
```

Comment:
API calls it `title`, so why renaming to `snap_title` on our side?
Why summary, description, etc don't have `snap_` prefix is that's what we go for?

Reply:
Yeah I thought about this, however I can imagine some cases where `title` would want to be the title of the page, in that case I'd want `snap_title` and `page_title` to clearly indicate what each title is. I want to avoid `title` sometimes being the page title and other times the title of the snap. 

Ideally I'd like everything to do with a snap to be namespaced under `snap` like `snap.title`, `snap.description` etc. but I don't think it's necessary.

Reply:
> I don't think it's necessary.

This is a scenario where it seems necessary. I will try to see if I can modify this in one of my refactor PR.